_posts/2024-02-27-comet-donation.md

```diff
@@ -0,0 +1,103 @@
+---
+layout: post
+title: "Announcing Apache Arrow DataFusion Comet"
+date: "2024-02-27 00:00:00"
+author: pmc
+categories: [release]
+---
+<!--
+{% comment %}
+Licensed to the Apache Software Foundation (ASF) under one or more
+contributor license agreements.  See the NOTICE file distributed with
+this work for additional information regarding copyright ownership.
+The ASF licenses this file to you under the Apache License, Version 2.0
+(the "License"); you may not use this file except in compliance with
+the License.  You may obtain a copy of the License at
+
+http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+{% endcomment %}
+-->
+
+# Introduction
+The Apache Arrow PMC is pleased to announce the donation of the [Comet project],
+a native Spark SQL Accelerator built on [Apache Arrow DataFusion].
+
+Comet is an Apache Spark Accelerator that uses Apache Arrow DataFusion to
+accelerate Spark workloads. It is written in Rust and designed as a drop-in
+replacement for Spark's JVM based SQL execution engine and offers significant
+performance improvements for some workloads as shown below.
+
+```text
```

Comment:
instead of text, maybe we can create some diagrams using tools like Lucidchart? I can help to translate this.

Reply:
That would be great if you can translate the diagram to an image. Maybe we could also (re)use such an image in the main comet readme.

Reply:
![Boson Vectorized Shuffle - Page 1](https://github.com/apache/arrow-site/assets/506679/90fc30c6-df97-4eef-b973-5b57aa98754a)

Reply:
How does this look? Let me know if you have ideas to improve it.

Reply:
I swapped the blog to use the image and I think it is looking pretty good

![Screenshot 2024-03-01 at 3 31 49 PM](https://github.com/apache/arrow-site/assets/490673/2fd1bafe-e226-43ea-808d-c1ce8e9b1873)

Reply:
Also I liked the image so much I propose putting it in the main comet repo: https://github.com/apache/arrow-datafusion-comet/pull/148